content/en/developers/ide_plugins/vscode/_index.md

```diff
@@ -152,6 +157,18 @@ Once the configuration file is created, the static analyzer runs automatically i
 
 <div class="alert alert-info">The Static Analysis feature does not require a Datadog account, as source files are analyzed locally.</div>
 
+## Exception Replay
+
+Exception Replay allows you to navigate through the stack trace frames of any Error Tracking code insight and get information about the values of the variables of the code running in production.
+
+In order to get access to this feature, you have to enable [Error Tracking Exception Replay][29] on Datadog.
```

Comment:
@rtrieu one of my teammates suggested to add `on Datadog` to avoid users thinking this is something we can enabled in the extension itself. WDYT?